docs_src/microservices/core/metadata/Ch-Metadata.md

```diff
@@ -212,6 +212,9 @@ Not all device services support dynamic discovery.  If it does support dynamic d
 
 ## Data Dictionary
 
+!!! edgey - "EdgeX 3.0"
+    **LastConnected** and **LastReported** are removed in EdgeX 3.0.
+
```

Comment:
should you also removed line 295 and 296?

Reply:
removed the properties